tests/src/Unit/Resolver/ChainCountryResolverTest.php

```diff
@@ -48,9 +48,9 @@ public function testResolver() {
     $third_resolver->expects($this->never())
       ->method('resolve');
 
-    $this->chainCountryResolver->addResolver($first_resolver);
-    $this->chainCountryResolver->addResolver($second_resolver);
-    $this->chainCountryResolver->addResolver($third_resolver);
+    $this->chainCountryResolver->addResolver($first_resolver, 0);
```

Comment:
The priority should be done as `-100`, `100`, and `200`. We should rebuild the container and properly test the priority sorting done when the container is rebuilt.